site/developer/model-testing/testing-overview.qmd

```diff
@@ -78,6 +78,8 @@ Next, find available tests and test suites using the {{< var vm.developer >}} or
 :::{#tests-explore}
 :::
 
+{{< video https://youtu.be/5J7wysDIXgI width="560" height="315" >}}
```

Comment:
@validbeck related to your visual tweak on the other video PR ... do we need to add something to our CSS for the drop shadow and rounded corners when we use the `{{< video >}}` directive?

Reply:
Yes, please copy and paste the code from the other PR and just change the URL for that here. I actually based that styling off of the video styling you did in the releases, it's just cleaned up a little.

Reply:
The other PR uses an iframe, but I'll see what I can do.

Reply:
Yes, you can just use the iframe here as well — I ended up replacing the quickstart video with that version of the code, for example: https://docs.validmind.ai/get-started/developer/try-with-jupyterhub.html

Reply:
Roger, I switched to an iframe with the same CSS tweaks for the video embed.